docs/AGENT_INTEGRATION_GUIDE.md

```
@@ -103,8 +100,14 @@ Headers:
 Content-Type: application/json
 X-Session-ID: <session-id>       # For context operations
 X-API-Key: <key>                 # If authentication is configured
+Authorization: Bearer <key>      # Alternative to X-API-Key
+Authorization: Basic <base64>    # Required for tool invocation when admin credentials are set
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)